lib/ansible_tower_client/job_template.rb

```diff
@@ -3,8 +3,15 @@ class JobTemplate
     extend CollectionMethods
     include InstanceMethods
 
-    def launch
-      Api.post("#{url}launch/")
+    attr_reader :response
+
+    def launch(vars = {})
+      launch_url = "#{url}launch/"
+      @response = Api.post(launch_url, vars).body
```

Comment:
@syncrou Sorry for the delay, I'm okay with merging this for now, but I feel like this should probably return a Job object.  What do you think?